dist/src/main/assembly/conf/brooklyn/default.catalog.bom

```diff
@@ -30,11 +30,11 @@ brooklyn.catalog:
         docker.policy.ha.enable: false
         docker.container.strategies:
         - $brooklyn:object:
-            type: clocker.docker.location.strategy.MaxContainersPlacementStrategy
+            type: clocker.docker.location.strategy.basic.MaxContainersPlacementStrategy
```

Comment:
No, the OSGi package rename was only a couple days ago, and involved a **lot** of changes. Things like: `brooklyn.clocker.docker.entity.container` getting split up and becoming `clocker.docker.entity`,  `clocker.docker.entity.container`,  `clocker.docker.entity.container.application`, `clocker.docker.entity.container.registry` and  `clocker.docker.entity.microservice`. Same for the Mesos packages, and so on. The placement strategy change here was just building on that.